### ISSUE TITLE
Reports ArtihmeticError for division, remainder and bitwise operations

### ISSUE THREAD

**Issue by marcospassos** [pepakriz/phpstan-exception-rules#77]
The following snippet reports a false positive:

```php
try {
    return $x / $y;
} catch (ArithmeticError $exception) {
}
```
```
ArithmeticError is never thrown in the corresponding try block  
```
[According to the PHP's docs](https://www.php.net/manual/en/class.arithmeticerror.php), division, remainder and bitwise operations can throw `ArithmeticException`.

**Comment by marcospassos** [comment]
@pepakriz what are your thoughts on this?

**Comment by pepakriz** [comment]
I see only two kinds of situations which can lead to `ArithmeticException`:

1) `intdiv(PHP_INT_MIN, -1);`
2) `1 << -3;`

Do you have an example with an arithmetic operator?
